tests/unit/utilities/find-addon-by-name-test.js

```diff
@@ -14,9 +14,17 @@ describe('findAddonByName', function() {
     }, {
       name: 'foo-bar',
       pkg: { name: 'foo-bar' },
+    }, {
+      name: 'foo-bar-baz',
+      pkg: { name: '@foo/bar-baz' },
     }];
   });
 
+  it('should return the foo-bar-baz addon when the unscoped package name is supplied', function() {
+    let addon = findAddonByName(addons, 'bar-baz');
+    expect(addon.name).to.equal('foo-bar-baz', 'should have found the foo-bar-baz addon');
```

Comment:
> In index.js the name is "prefix-my-package-name".

The fact that there is a difference here seems bad, as described the following two packages`@foo/bar` and `bar` will collide and cause grief.